YoCode/Program.cs

```diff
@@ -1,23 +1,53 @@
 ﻿using System;
+using System.Linq;
 using System.IO;
 
 namespace YoCode
 {
     public class Program
     {
+
         static void Main(string[] args)
         {
+            var modifiedTestDirPath = args[0];
+            var originalTestDirPath = args[1];
+
+            var consoleOutput = new PrintToConsole();
+            var testResults = new TestResults();
+            var fileReader = new FileImport();
+
+            var modifiedTest = fileReader.GetAllFilesInDirectory(modifiedTestDirPath);
+            var originalTest = fileReader.GetAllFilesInDirectory(originalTestDirPath);
+
+            var dir = new Directory(originalTest, modifiedTest);
+
+            if (FileChangeChecker.ProjectIsModified(dir))
+            {
+                testResults.AnyFileChanged = true;
+                // UI test
+                var keyWords = new string[] { "miles", "kilometers", "km" };
+                var modifiedHtmlFiles = dir.GetFilesInDirectory(modifiedTestDirPath, FileTypes.html).ToList();
+
+                UICheck uiChecker = new UICheck(modifiedHtmlFiles, keyWords);
+
+                testResults.Lines = uiChecker.ListOfMatches;
+
+                // Solution file exists
+                testResults.SolutionFileExist = dir.GetFilesInDirectory(modifiedTestDirPath, FileTypes.sln).Count() != 0;
 
-            Console.WriteLine("Welcome to YoCode!");
+                // Git repo used
+                GitCheck gitChecker = new GitCheck();
+                gitChecker.ExecuteTheCheck();
+                testResults.GitUsed = gitChecker.GitUsed;
+                // Will not work on different systems due to issues in GitUsed method
+            }
+            else
```

Comment:
Braces. Even though you don't technically need it here, it's good to have the `{}` around this `else` code because it makes it easier to notice if you're quickly looking through the code. Also, without the `{}` it is quite easy to accidentally break the `else` if somebody is moving code around (it's more fragile without the `{}`).

Reply:
Here's an "interesting" example of the brace problem https://embeddedgurus.com/barr-code/2014/03/apples-gotofail-ssl-security-bug-was-easily-preventable/ 

(although this kind of thing is also easy to find with static analysis, so I think the readability of it is a better argument)